lib/std/child_process.zig

```diff
@@ -372,7 +372,7 @@ pub const ChildProcess = struct {
 
     fn waitUnwrapped(self: *ChildProcess) !void {
         const res: os.WaitPidResult = res: {
-            if (builtin.os.tag == .linux and self.request_resource_usage_statistics) {
+            if ((builtin.os.tag == .linux or is_darwin) and self.request_resource_usage_statistics) {
```

Comment:
I think a switch here will be more readable here than a long `if` statement like this. Something like

```zig
if (self.request_resource_usage_statistics) {
    switch (builtin.os.tag) {
        .linux => { },
        .macos, .ios => { },
        else => { },
    }
}
```